lib/ansible/module_utils/basic.py

```diff
@@ -1983,29 +1983,15 @@ def atomic_move(self, src, dest, unsafe_writes=False):
                             e = get_exception()
                             if e.errno != errno.EPERM:
                                 raise
-                        os.rename(b_tmp_dest_name, b_dest)
+                        try:
```

Comment:
The answer to that is in the commit message: "If the code fails in shutil.copy2 or shutil.move this should not be related to issue #13981 since they write to b_tmp_dest_name."